src/pages/defi/uniswap-v2-add-remove-liquidity/UniswapV2Liquidity.sol

```diff
@@ -13,10 +14,20 @@ contract TestUniswapLiquidity {
         uint _amountB
     ) external {
         IERC20(_tokenA).transferFrom(msg.sender, address(this), _amountA);
-        IERC20(_tokenB).transferFrom(msg.sender, address(this), _amountB);
+        // IERC201(_tokenB).transferFrom(msg.sender, address(this), _amountB);
```

Comment:
1. Create a internal function called `safeTransferFrom`
2. Use `safeTransferFrom` for  both tokenA and B
3. Comment why `safeTransferFrom` is needed